Tools/WinMLRunner/src/OutputHelper.h

```diff
@@ -518,16 +540,18 @@ class OutputHelper
     }
     void SaveBindTimes(Profiler<WINML_MODEL_TEST_PERF>& profiler, uint32_t iterNum)
     {
-        m_clockBindTimes[iterNum] = profiler[BIND_VALUE].GetClockTime();
+        m_clockBindTimes[iterNum] =
+            (iterNum == 0) ? profiler[BIND_VALUE_FIRST_RUN].GetClockTime() : profiler[BIND_VALUE].GetClockTime();
     }
     void SaveEvalPerformance(Profiler<WINML_MODEL_TEST_PERF>& profiler, uint32_t iterNum)
     {
-        m_clockEvalTimes[iterNum] = profiler[EVAL_MODEL].GetClockTime();
-        m_CPUWorkingDiff[iterNum] = profiler[EVAL_MODEL].GetCpuWorkingDiff();
-        m_CPUWorkingStart[iterNum] = profiler[EVAL_MODEL].GetCpuWorkingStart();
-        m_GPUSharedDiff[iterNum] = profiler[EVAL_MODEL].GetGpuSharedDiff();
-        m_GPUSharedStart[iterNum] = profiler[EVAL_MODEL].GetGpuSharedStart();
-        m_GPUDedicatedDiff[iterNum] = profiler[EVAL_MODEL].GetGpuDedicatedDiff();
+        iterationNum = (iterNum == 0) ? EVAL_MODEL_FIRST_RUN : EVAL_MODEL;
+        m_clockEvalTimes[iterationNum] = profiler[EVAL_MODEL_FIRST_RUN].GetClockTime();
+        m_CPUWorkingDiff[iterationNum] = profiler[EVAL_MODEL_FIRST_RUN].GetCpuWorkingDiff();
+        m_CPUWorkingStart[iterationNum] = profiler[EVAL_MODEL_FIRST_RUN].GetCpuWorkingStart();
```

Comment:
profiler[EVAL_MODEL_FIRST_RUN] will always return the performance of the first evaluate. Is this intended?